app/Models/GameHash.php

```diff
@@ -54,7 +57,8 @@ public function getRouteKeyName(): string
     public function getActivitylogOptions(): LogOptions
     {
         return LogOptions::defaults()
-            ->logOnlyDirty();
+            ->logOnlyDirty()
+            ->dontSubmitEmptyLogs();
```

Comment:
I'm not seeing any new entries in the `audit_log` table.

Reply:
Good catch -- updated in latest.